app/controllers/questions_controller.rb

```diff
@@ -67,7 +68,9 @@ def create
   def update
     set_wip
     if @question.update(question_params)
-      Newspaper.publish(:question_update, { question: @question }) if @question.saved_change_to_wip?
+      wip_changed = @question.saved_change_to_wip?
+      ActiveSupport::Notifications.instrument('question.update', { question: @question }) if wip_changed
+      Newspaper.publish(:question_update, { question: @question }) if wip_changed
```

Comment:
後置のifは1行の時の書き方なのでこの場合は下記でいいかもです。

```suggestion
      if @question.saved_change_to_wip?
        ActiveSupport::Notifications.instrument('question.update', { question: @question })
        Newspaper.publish(:question_update, { question: @question })
      end
```

Reply:
後置ifを通常のif文に修正いたしました。
コミット：https://github.com/fjordllc/bootcamp/pull/8987/commits/005b9e86e35c3c95bcb06de8c6a92ae1a3ec762b

※rebaseの影響で該当のレビューコメントが表示されなくなっているようなので、こちらに返信させていただきました。🙇‍♀️